_sass/base/_markdown.scss

```diff
@@ -26,13 +26,14 @@
   // List Items
   li {
     line-height: 24px;
+    margin: 5px;
   }
 
   // Lists
   ol,
   ul {
     list-style: initial;
-    font-size: 15px;
+    font-size: 16px;
```

Comment:
Can we fix this to not specify absolute font sizes in pixels? This doesn't work well on screens of higher density. Please use one of the SASS variables defined in _sass/base/_config.scss instead. I've been trying to stomp out these absolute pixel values over time, so if we're changing things, let's change them for the better.